README.md

```diff
@@ -0,0 +1,50 @@
+# ansible-vault role
+Installs and configures Hashicorp Vault for Ubuntu trusty/xenial servers. It also sets AWS, LDAP/Jumpcloud and User&Password authentication methods. It uses Consul as the Storage backend.
```

Comment:
In fact, I think this is not only for trusty/xenial. It should work with any `systemd` flavor.

Reply:
Great, I'll update it.